### PR TITLE
adding HiCBrowser 1.0 recipe

### PR DESCRIPTION
**Pull request by fidelram**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/hicbrowser/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package:
+  name: hicbrowser
+  version: '1.0'
+
+source:
+  fn: 1.8.tar.gz
```
Comment (bgruening) [inline recipes/hicbrowser/meta.yaml]: Can you change this to: 1.0.tar.gz

File: recipes/hicbrowser/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    - flask >=0.10.1
+    - hicexplorer >=1.7
+    - bx-python
+
+test:
+  imports:
+    - hicbrowser
+
+  commands:
+    - runBroser --version
```
Comment (bgruening) [inline recipes/hicbrowser/meta.yaml]: This seems to be a typo?

File: recipes/hicbrowser/meta.yaml (hunk 1 above)
Comment (fidelram) [reply recipes/hicbrowser/meta.yaml]: oops (copy & paste error)

File: recipes/hicbrowser/meta.yaml (hunk 2 above)
Comment (fidelram) [reply recipes/hicbrowser/meta.yaml]: indeed

**Comment by bgruening** [comment]
@fidelram fixed and merged!

**Pull request merged**